scrapinghub/client.py

```diff
@@ -203,6 +195,17 @@ def _extract_spider_id(self, params):
             raise ValueError('Please use same spider id')
         return spiderid
 
+    def update_tags(self, add=None, remove=None, spidername=None):
+        spidername = spidername or (self.spider.name if self.spider else None)
+        if not spidername:
+            raise ValueError('Please provide spidername')
```

Comment:
@chekunkov Makes sense to me, I'd move the checks to Dash then, maybe someday we add scheduling a default/last-launched spider, and updating tags for a whole project will also be allowed.

For notes now Dash validates spidername on scheduling jobs pretty good, but it expects `spider` param while client `schedule()` accepts `spidername`, I can rename it in the client or we should map it in the Dash:
```
{"spider":["This field is required."],"status":"error","message":"spider: This field is required."}
```
As for updating tags without specifying a spider - it's less clear, there should be a check:
```
{"message": "No query filters provided", "status": "badrequest"}
```